x-pack/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -107,7 +107,7 @@ export const TransactionActionMenu: FunctionComponent<Props> = ({
               event.preventDefault();
               core.application.navigateToApp(app, {
                 path: `${rest.join('/')}${
-                  parsed.search ? `&${parsed.search}` : ''
+                  parsed.search ? `${parsed.search}` : ''
```

Comment:
Was there a reason for the ampersand or was it a type? Wondering if we are just moving the problem to another case.

Would it make sense to add a small test for this?

Reply:
Maybe the ampersand was there as an URL parameter, so I'd go with @sqren's approach of unit testing the generated path.

Taking a quick look at the original bug, where the `&` is appended, it seems that the `${rest.join('/')}` includes `kubernetes.pod.name` or `container.id` as part of the URL, not its parameters, so I guess that the `&` is there as a valid parameter separator.

Reply:
ack, I've added some tests.